standards/core/peer-exchange.md

```diff
@@ -66,26 +66,40 @@ message PeerInfo {
   bytes enr = 1;
 }
 
-message PeerExchangeQuery {
+message PeerExchangeRequest {
   uint64 num_peers = 1;
 }
 
 message PeerExchangeResponse {
   repeated PeerInfo peer_infos = 1;
+  uint32 status = 10;
+  optional string desc = 11;
 }
 
 message PeerExchangeRPC {
-  PeerExchangeQuery query = 1;
+  PeerExchangeRequest request = 1;
   PeerExchangeResponse response = 2;
 }
 
 ```
 
 The `enr` field contains a Waku ENR as specified in [WAKU2-ENR](./enr.md).
 
-Requesters send a `PeerExchangeQuery` to a peer.
+Requesters send a `PeerExchangeRequest` to a peer.
 Responders SHOULD include a maximum of `num_peers` `PeerInfo` instances into a response.
 Responders send a `PeerExchangeResponse` to requesters containing a list of `PeerInfo` instances, which in turn hold an ENR.
+Responders, MUST include a `status` in the response in any case and possible descriptive form of the problem if any.
+
+### Possible status codes
+
+| Result | Code | Note |
+|--------|------|------|
+| SUCCESS  | 200 | Successfull request-respond. In response the answer must contain `PeerExchangeResponse`  |
+| BAD_REQUEST | 400   | Wrong request payload. It must only contain `reques` field.  |
+| BAD_RESPOND | 401   | Wrong respond payload. If success it must contain `respond` and `response_status` fields. If failure, only `response_status` is set.  |
```

Comment:
```suggestion
| BAD_RESPONSE | 401   | Wrong response payload. If success it must contain `response` and `response_status` fields. If failure, only `response_status` is set.  |
```

Unless this is for compatibility elsewhere, "respond" should be "response" in all field names. "Respond" is the verb, with "response" being the noun.

Reply:
Absolutely right, fixing it.